test/test.c

```diff
@@ -34127,6 +34238,7 @@ void test_MicroBasics(void)
         testCond(
             (NATS_OK == nats_JSONGetStrPtr(array[0], "name", &str)) && (strcmp(str, "do") == 0)
             && (NATS_OK == nats_JSONGetStrPtr(array[0], "subject", &str)) && (strcmp(str, "svc.do") == 0)
+            && (NATS_OK == nats_JSONGetStrPtr(array[0], "queue_group", &str)) && (strcmp(str, "q") == 0)
```

Comment:
In case you did not "replace all", this would be one place to use `MICRO_DEFAULT_QUEUE_GROUP`.